feat: 🇵🇰 pakistan feed

melMass opened the pull request.

melMass committed:
feat: ⚡️ template for the upcoming contract

LeonNicholls commented on src/pages/search/index.js:
```diff
@@ -316,16 +316,17 @@ export class Search extends Component {
     tags: [
-      { id: 11, value: '🇺🇦 ukraine', label: 'ukraine' },
-      { id: 12, value: '🏳️‍🌈 tezospride', label: 'tezos pride' },
+      { id: 10, value: 'pakistan', label: '🇵🇰 pakistan' },
```
These are used as aria-label values for accessibility. The emoticons don't work with the text-to-speech systems, so should be avoided. Just keep textual info for the labels.

melMass replied:
We could either use the value or another key, but the label/value was used the other way around. This also avoids emoticons check in ifs

melMass replied:
@NoRulesJustFeels fixed in the last commit

melMass committed:
fix: ✨ use aria key

melMass committed:
fix: ⚡️ use the real address

melMass committed:
fix: ⚡️ ban filter

This is solved in develop, there was no
filtering on half the feeds...

melMass committed:
feat: ⚡️ store the current tab in states

The `prev` state was unused.
I added a `current` one that simply stores the value of the current
tag.
This is then used to display a simple notice on the fund feeds for
instance.

melMass committed:
fix: ⚡️ mobile style & new text

melMass committed:
fix: ⚡️ text layout

melMass committed:
fix: ⚡️ fix banner

melMass committed:
fix: ⚡️ infinite scroll

melMass committed:
fix: ⚡️ fetch 20 per batch

xat reviewed (approved):
don't have the time to properly review. but merch said, that he manually tested it. so merging it now.
